hydra-plutus/src/Hydra/Contract/HeadTokens.hs

```diff
@@ -52,10 +52,11 @@ validate initialValidator headValidator seedInput action context =
 validateTokensMinting :: ValidatorHash -> ValidatorHash -> TxOutRef -> ScriptContext -> Bool
 validateTokensMinting initialValidator headValidator seedInput context =
   traceIfFalse "minted wrong" $
-    participationTokensAreDistributed currency initialValidator txInfo nParties
+    seedInputIsConsumed
+      -- TODO remove below checks and Init mutations
```

Comment:
this will be addressed once the off-chain logic is tested.

this is also needed to be removed in order to generate valid cluster specs.